milestones/gain_awareness_of_ansible_meetups.md

```diff
@@ -0,0 +1,128 @@
+## What is an Ansible Meetup?
+
+Ansible meetup is a gathering of a small to medium-sized group of Systems Engineers, DevOps and SREs. Where they talk about their Automation journey focusing on Ansible, it is a place to teach and learn from each other. Meetup provides a platform for upstream and downstream to meet and share ideas.
+
+## How to become a member of the meetup?
+
+One can be a member of an Ansible meetup group by:
+
+### Attending the meetups
+
+Check the list of current Ansible meetups. If there is already a meetup in your area, join it and register for an upcoming event.
+
+### Presenting at the meetup
+
+Do you want to present at an Ansible meetup? Do you have an idea for a session? Feel free to reach out to the meetup group either by
+
+- Commenting on the meetup page itself.
+- Contacting the organizers via meetup.com direct messages.
+- Open an issue at the [Github repo of the meetup group](https://github.com/ansible-community/?q=meetup) (if available).
+- Share your idea at [#ansible-social](https://matrix.to/#/#social:ansible.com) Matrix room or one of the [regional rooms](https://matrix.to/#/#ansible-regional:matrix.org).
+
+
+Meetup organizers always look for good presenters to enrich their events and meetup groups. So do not hold back from sharing your ideas.
+
+### Hosting a meetup
+
+Want to host a meetup at your organization? Then please reach out
+ - by commenting on a meetup page of a local meetup
+ - via social media
+ - send us an [email](meetup@ansible.com)
+
+### Organizing a meetup
+
+No Ansible meetup in your area? Send us an [email](meetup@ansible.com) introducing yourself. Be sure to include a link to your profile at meetup.com.
+
+We want to be sure that meetup organizers are committed to building a strong local community of Ansible users. After a brief discussion, we will create the Ansible meetup group in your area, appoint you as a co-organizer, and announce your new group to the world.
+
+Secure a date and location for your first meetup! We will help you by spreading the word on social media, reaching out to potential speakers and sponsors in your area, and suggesting new and interesting Ansible content on a regular basis.
+
+## Who can become an organizer?
+
+Anyone from the wider Ansible community who is committed to run an Ansible meetup group and plan/organize meetups can become an organizer. The fundamental understanding of the Open Source community, its working, and the belief in this ethos is essential to be an efficient community builder. Some knowledge about Ansible is always a plus.
+
+## Ansible Community Code of Conduct
```

Comment:
Could this be a link to the [CoC hosted in the main docsite](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html)? Pointing there would mean less maintenance and less chance of "content drift" (where one copy of the CoC gets updated but others don't).